### PR TITLE
Add contribution and development docs

### PR DESCRIPTION
**Pull request by kentakozuka**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 Fixes #
 
 **Does this PR introduce a user-facing change?**:
+- **Who does this affect**:
+- **How to migrate**:
+- **Why make this breaking change**:
```
Comment (khanhtc1202) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion

- **Is this breaking change**:
- **How to migrate (if breaking change)**:
- **How are users affected by this change**:

```
And we can remove the below `release-note` section/block

File: docs/README.md [hunk 2]
```diff
@@ -6,6 +6,32 @@ The source files for the documentation is placing in [content](https://github.co
...
+3. Update docs that are related to the latest released docs version:
+- Change the docs' content that fixes the issue under `/docs-dev` and `/docs-v0.x.x`, they share the same file structure so you should find the right files in both directories.
+- Use `make gen/stable-docs` command to sync the latest released version docs under `/docs-v0.x.x` to `/docs`
+
+If you find any issues related to the docs, we're happy to accept your help.
+
+# How to run website locally
+
+## Prerequisite
+- [Hugo 0.92.1+extended](https://gohugo.io/)
```
Comment (kentakozuka) [inline docs/README.md:34]: `+extended` added 👍

File: cmd/piped/README.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
+# Pipd Agent
```
Comment (khanhtc1202) [inline cmd/piped/README.md:1]:
```suggestion
# Piped Agent
```

File: cmd/pipecd/README.md [hunk 4]
```diff
@@ -1,3 +1,47 @@
+# PipeCD Controle Plane
 ## Development
 
-See [this documentation](https://pipecd.dev/docs/contribution-guidelines/control-plane-development/).
+## Prerequisites
+
+- [Go 1.19](https://go.dev/)
```
Comment (khanhtc1202) [inline cmd/pipecd/README.md:6]:
```suggestion
- [Go 1.19 and later](https://go.dev/)
```

File: cmd/piped/README.md [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
 ## Development
 
-See [this documentation](https://pipecd.dev/docs/contribution-guidelines/piped-development/)
+## Prerequisites
+
+- [Go 1.19](https://go.dev/)
```
Comment (khanhtc1202) [inline cmd/piped/README.md:6]:
```suggestion
- [Go 1.19 or later](https://go.dev/)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+If you want to work on any of these issues, simply leave a message saying "I'd like to work on this," and we will assign the issue to you and update its status as "claimed." We expect you to submit a pull request within seven days so that we can assign the issue to someone else if you are unavailable.
+
+## Development
+
+PipeCD consists of several components and docs:
+
+- **cmd/pipecd**: A centralized component that manages deployment data and provides a gRPC API for connecting pipeds, as well as web functionalities such as authentication.
+- **cmd/piped**: piped is an agent component that runs in your cluster.
+- **cmd/pipectl**: The command-line tool for PipeCD.
+- **cmd/launcher**: The command that enables remote upgrade of the piped agent.
```
Comment (khanhtc1202) [inline CONTRIBUTING.md:78]:
```suggestion
- **cmd/launcher**: The command executor that enables the remote upgrade feature of the piped agent.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+We have various integration systems that run automated tests to prevent mistakes. The maintainers will also review your code and fix obvious issues. These systems are in place to minimize your worries about the process. Your code contributions are more important than adhering to strict procedures, although completing the checklist will undoubtedly save everyone's time.
+
+### Commit Messages
+Commit messages should be simple and use easy words that indicate the focus of the commit and its impact on other developers. Summary in the present tense. Use capital case in the first character but do not use title case.
+
+Example
+
+```
+Add imports to terraform plan result
```
Comment (khanhtc1202) [inline CONTRIBUTING.md:112]:
```suggestion
Add imports to Terraform plan result
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+```
+
+Don't stress too much about PR titles. The maintainers will help you get the title right.
```
Comment (khanhtc1202) [inline CONTRIBUTING.md:115]: ❤️

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+### Breaking Changes
+
+When introducing a new breaking change, follow the pull request template and make sure to complete the following sections:
+
+```md
+**Does this PR introduce a user-facing change?**:
+- **Who does this affect**:
+- **How to migrate**:
+- **Why make this breaking change**:
```
Comment (khanhtc1202) [inline CONTRIBUTING.md:154]:
```suggestion
- **Is this breaking change**:
- **How to migrate (if breaking change)**:
- **How are users affected by this change**:
```
reflect above changed

**Comment by khanhtc1202** [comment]
@kentakozuka Commented, overall looks amazing. Nice job 🙌

**Comment by kentakozuka** [comment]
@khanhtc1202 Thanks! Updated 👍

**Review by khanhtc1202 — APPROVED** [review]
Amazing! Thank you 🚀

**Pull request merged**